src/components/DMsList/DMsListStyled.js

```diff
@@ -62,24 +92,48 @@ export const BadgeStyled = styled.div`
 export const ChatName = styled.div`
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
-  width: 116px;
+  justify-content: center;
+  gap: 8px;
+  width: calc(100% - 58px);
   height: 50px;
-  & > p {
+`;
+
+export const NameAndDayBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  p {
+    font-size: 14px;
+    font-weight: 400;
+    flex-shrink: 0;
     color: var(--color-grey-9);
+    white-space: nowrap;
   }
-  @media ${device.laptop} {
-    width: 156px;
-  }
 `;
 
-export const MessageDay = styled.div`
+export const CompanionName = styled.h6`
+  flex-grow: 1;
+  margin-right: 8px;
+  font-size: 18px;
+  font-weight: ${({ $isActive }) => ($isActive ? '600' : '400')};
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+export const MessageAndCountBox = styled.div`
   display: flex;
-  align-items: flex-start;
-  justify-content: flex-end;
-  width: 40px;
-  height: 50px;
-  & > p {
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  gap: 8px;
+  p {
     color: var(--color-grey-9);
+    font-size: 14px;
```

Comment:
We already have font-size: 14px for the p tag in the GlobalStyles.js.

Reply:
done